### ISSUE TITLE
this.isMounted is not a function

### ISSUE THREAD

**Issue by manuelmazzuola** [BinaryMuse/fluxxor#138]
When `StoreWatchMixin` calls `_setStateFromFlux` I had this

```
Unhandled rejection TypeError: this.isMounted is not a function
    at StoreWatchMixin._setStateFromFlux (http://localhost:3000/js/app.js:655:15)
   ...
```

Because of this https://facebook.github.io/react/docs/component-api.html#ismounted
`This method is not available on ES6 class components that extend React.Component. It may be removed entirely in a future version of React.`

**Comment by manuelmazzuola** [comment]
This will resolve the issue BinaryMuse/fluxxor/pull/135

**Comment by BinaryMuse** [comment]
Closed by #135
